pavelib/assets.py

```diff
@@ -64,6 +64,9 @@
     'cms': [path('lms') / 'static' / 'sass' / 'partials', ],
 }
 
+# Collectstatic log directory setting
+COLLECTSTATIC_LOG_DIR_ARG = "collect_log_dir"
```

Comment:
I thought the point was the push the logic further up the stack? That's what I was attempting to accomplish in the latest set of changes.

The global string variable was just to avoid duplication. I used it in the tests as well so the tests can depend on that same global setting. I can remove the global nature of it, though, if you feel strongly about it.